### PR TITLE
Add performance logging documentation

### PR DESCRIPTION
**Pull request by JasonDing0401**

### CODE REVIEW

**Comment by parasj** [comment]
The build is [failing](https://readthedocs.org/projects/skyplane/builds/17980140/) but link this page into the website, update index.rst to add it to the table of contents at the bottom of the page.

**Comment by JasonDing0401** [comment]
Fixed! It's ready for review now.

File: docs/usage_stats_collection.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Here are the guiding principles of our collection policy:
+
+- **No surprises** — you will be notified before we begin collecting data. You will be notified of any changes to the data being collected or how it is used.
+- **Easy opt-out:** You will be able to easily opt-out of data collection.
+- **Transparency** — you will be able to review all data that is sent to us.
+- **Control** — you will have control over your data, and we will honor requests to delete your data.
+- We will **not** collect any personally identifiable data or proprietary code/data.
+- We will **not** sell data or buy data about you.
+
+## What data is collected
```
Comment (parasj) [inline docs/usage_stats_collection.md:14]: Explicitly list everything we collect here as bullet points. We shouldn't route people to `UsageStatsToReport` but instead also point them to `/tmp/...`

File: docs/usage_stats_collection.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Usage Stats Collection
```
Comment (parasj) [inline docs/usage_stats_collection.md:1]: Rename mentions of "usage" to "performance". E.g. the title should be "Performance stats". Also update this in the body.

File: docs/usage_stats_collection.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Skyplane collects usage stats by default. All data is anonymous and helps us maintain an open-source map of cloud network performance data. This data is used to improve the speed of your transfers by routing your traffic around slow links. It will only be used by the Skyplane team to improve the services and for research purposes.
```
Comment (parasj) [inline docs/usage_stats_collection.md:3]: `Skyplane collects usage stats by default` => `Skyplane collects performance statistics. Some of this data is shared with the Skyplane developers to improve the accuracy of the Skyplane solver.`

File: docs/index.rst [hunk 4]
```diff
@@ -41,6 +41,7 @@ Contents
    configure
    architecture
    faq
+   usage_stats_collection
```
Comment (parasj) [inline docs/index.rst:44]: Reorder so the FAQ is last.

**Review by parasj — APPROVED** [review]
Some minor changes requested, then please merge.

**Pull request merged**